notebooks/exploration/utils/rating.py

```diff
@@ -0,0 +1,145 @@
+import pandas as pd
+from collections import defaultdict
+import math
+from copy import deepcopy
+import numpy as np
+import pandas as pd
+from collections import defaultdict
+import math
+from copy import deepcopy
+import numpy as np
+import trueskill
+from scipy.stats import norm as sc_norm
+
+
+class Rater:
```

Comment:
Could you add some doc strings for these functions? I'm a bit new to this stuff so would help me to follow. Seems like this could be moved to src if final?